<launch>
  <arg name="gui" default="true" />
  <arg name="model" default="mask_rcnn_fpn_resnet101"/>
  <arg name="pretrained_model" default="coco"/>
  <arg name="gpu" default="-1"/>
  <arg name="label_names"
       default="$(find ros_chainercv)/yaml/coco_instance_segmentation_label_names.yaml" />

  <include file="$(find ros_chainercv)/sample/sample_instance_segmentation.launch">
    <arg name="gui" value="$(arg gui)" />
    <arg name="model" value="$(arg model)" />
    <arg name="pretrained_model" value="$(arg pretrained_model)" />
    <arg name="gpu" value="$(arg gpu)" />
    <arg name="label_names" value="$(arg label_names)" />
  </include>

</launch>
